js/defaultKeybindings.js

```diff
@@ -256,22 +256,17 @@ const defaultKeybindings = {
       lastReload = time
     })
 
+    keybindings.defineShortcut('reloadIgnoringCache', function () {
+        webviews.callAsync(tabs.getSelected(), 'reloadIgnoringCache')
+    })
+
     keybindings.defineShortcut({ keys: 'mod+=' }, function () {
       webviewGestures.zoomWebviewIn(tabs.getSelected())
     })
 
     keybindings.defineShortcut('showHistory', function () {
       tabEditor.show(tabs.getSelected(), '!history ')
     })
-
-    // reload the webview when the F5 key is pressed
-    document.body.addEventListener('keydown', function (e) {
-      if (e.keyCode === 116) {
```

Comment:
This was previously just the standard `f5` key; I think what we want to do is add `f5` to the `reload` shortcuts list in defaultKeyMap to match the previous behavior, and then keep `mod+f5` as `reloadIgnoringCache`.

Reply:
Done.